tests/test_tasks.py

```diff
@@ -1,118 +1,137 @@
 import asyncio
-import logging
-import time
-from asyncio import AbstractEventLoop
-from typing import TYPE_CHECKING, Any, Dict, List, NoReturn, Tuple
+from typing import NoReturn
 
 import pytest
-from pytest import LogCaptureFixture
-
-if TYPE_CHECKING:
-    from pytest.capture import CaptureFixture
-else:
-    CaptureFixture = Any
+from pytest_mock import MockerFixture
 
 from fastapi_restful.tasks import repeat_every
 
-logging.basicConfig(level=logging.INFO)
 
+@pytest.mark.asyncio
+async def test_repeat_every_with_sync_function_and_max_repetitions(mocker: MockerFixture) -> None:

```

Comment:
at this point I think it will be better to shorten the name and just doc the test - something like 'test_repeat_every_synced_and_maxed'?  it just seems as if you are using test names in code and it makes it less readable

Reply:
Actually I read that tests, in contrast to normal functions, should be as declarative as possible.
@yuval9313 What do you think?